modules/ROOT/pages/compatibility-issues-runtime-java.adoc

```diff
@@ -0,0 +1,15 @@
+= Troubleshooting Issues with Mule Runtime and Java
```

Comment:
This topic has this in the nav: Resolve Compatibility Issues

Reply:
This file belongs to the troubleshooting section.
* xref:troubleshooting-studio.adoc[Troubleshooting]
 ** xref:faq-default-browser-config.adoc[Troubleshoot Issues with Platform Content in Studio]
  *** xref:studio-xulrunner-wx-task.adoc[Windows]
  *** xref:studio-xulrunner-lnx-task.adoc[Linux]
  *** xref:studio-xulrunner-unx-task.adoc[MacOS]
 ** xref:compatibility-issues-runtime-java.adoc[Resolve Compatibility Issues]
 Or am I misunderstanding your question?

Reply:
Oh, OK. I misread the nav.

Reply:
OK! Thanks!